__test__/api/admins/getSingleAdmin.test.js

```diff
@@ -0,0 +1,63 @@
+const app = require("../../../server");
+const supertest = require("supertest");
+const mongoose = require("mongoose");
+const AdminModel = require("../../../models/admins");
+const { getOrgToken } = require("../../../utils/auth/tokenGenerator");
+const { validateOrg } = require("../../../utils/validationRules/admins/getSingleAdminSchema");
+
+const request = supertest(app);
+
+describe("READ /v1/admins/get-single-admin", () => {
+  let adminInfo = {
+    adminId: "adminIdStr",
+  };
+  let url = `/v1/admins/get-single-admin`;
```

Comment:
This url does not match anyone we have in the docs for this route
[See Docs](https://comment.microapi.dev/#/Admins/get_admins__adminId_)